e2e/transactions/transactions_test.go

```diff
@@ -811,3 +811,103 @@ func (s *TransactionsTestSuite) TestEvictionOfQueuedTransactions() {
 	m.Unlock()
 	s.Zero(txQueue.Count(), "transaction count should be zero: %d", txQueue.Count())
 }
+
+func (s *TransactionsTestSuite) TestCompleteMultipleQueuedTransactionsUpstream() {
```

Comment:
looks good, but please make it DRY :)

Reply:
Done, added `setupUpstreamNode`, `setupLocalNode` and `sendConcurrentTransactions`, PTAL.